modules/roles/server/config/routes.js

```diff
@@ -1,20 +1,32 @@
 module.exports = [
-  {
-    route: '/roles',
-    type: 'POST',
-    method: 'create',
-    secure: true
-  },
-  {
-    route: '/roles',
-    type: 'PUT',
-    method: 'update',
-    secure: true
-  },
-  {
-    route: '/roles/:id',
-    type: 'DELETE',
-    method: 'delete',
-    secure: true
-  } 
+{
+  route: '/roles',
+  type: 'POST',
+  method: 'create',
+  secure: false
```

Comment:
ew, was this for testing?  It looks like all of the role endpoints are insecure.